ecl/hqlcpp/hqlcpp.cpp

```diff
@@ -11943,7 +11945,16 @@ void HqlCppTranslator::buildFunctionDefinition(IHqlExpression * funcdef)
         if (languageAttr)
             buildScriptFunctionDefinition(funcctx, funcdef, proto);
         else
-            buildCppFunctionDefinition(funcctx, bodyCode, proto);
+        {
+            bool isInline = bodyCode->hasAttribute(inlineAtom);
+            if (isInline)
```

Comment:
This also needs to be conditional on 
(translator.queryOptions().spanMultipleCpp - otherwise if you are generating into a single c++ file the function never appears.